src/utilities.ts

```diff
@@ -1,16 +1,27 @@
+import {
+  JSONValue,
+  parse,
+} from "https://deno.land/std@0.149.0/encoding/jsonc.ts";
+
+// Export alias to avoid having to import the above elsewhere
+export type JSONCValue = JSONValue;
```

Comment:
Rather than import the `JSONValue` type into three separate files from the `.jsonc.ts` module, I've alias'd and re-exported it for use from this file. If this isn't an acceptable or preferable way to go about it, let me know.

Reply:
Good call, centralizing deps somewhere at least helps us keep things using the same version. 

Another convention that i[s suggested](https://deno.land/manual/examples/manage_dependencies#overview) for deno libraries is to use a `deps.ts` file to import/export shared deps like this. We have a `dev_deps.ts` already for test-related dependencies, so we could add a `deps.ts` file for this dependency.

Reply:
Perf. I'll move this to `deps.ts`.